lib/bas/bot/fetch_images_from_discord.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require_relative "./base"
+require_relative "../read/default"
+require_relative "../utils/discord/request"
+require_relative "../write/postgres"
+
+module Bot
+  ##
+  # The Bot::FetchImagesFromDiscord class serves as a bot implementation to read images
+  # from a any thread of Discord channel.
+  #
+  # <br>
+  # <b>Example</b>
+  #
+  #   options = {
+  #     process_options: {
+  #       secret_token: "discord_bot_token"
+  #       discord_channel: "discord_channel_id"
+  #     },
+  #     write_options: {
+  #       connection: {
+  #         host: "localhost",
+  #         port: 5432,
+  #         dbname: "bas",
+  #         user: "postgres",
+  #         password: "postgres"
+  #       },
+  #       db_table: "review_media",
+  #       tag: "FetchImagesFromDiscord"
+  #     }
+  #   }
+  #
+  #   bot = Bot::FetchImagesFromDiscord.new(options)
+  #   bot.execute
+  #
+  class FetchImagesFromDiscord < Bot::Base
+    # read function to execute the PostgresDB Read component
+    #
+    def read
+      reader = Read::Default.new
+
+      reader.execute
+    end
+
+    # Process function to execute the Discord utility to fetch images from a discord channel threads
```

Comment:
What do you think about changing it to `Process function to execute the Discord utility to fetch images from Discord channel threads.`? :thinking: